fsspec_reference_maker/hdf.py

```diff
@@ -164,8 +175,9 @@ def translator(self, name, h5obj):
             if self._xr:
                 # Do this for xarray...
                 adims = self._get_array_dims(h5obj)
-                za.attrs['_ARRAY_DIMENSIONS'] = adims
-                lggr.debug(f'_ARRAY_DIMENSIONS = {adims}')
+                if adims:
```

Comment:
else?

Reply:
Is there a need for empty _ARRAY_DIMENSIONS_ attribute? My understanding is that no need for _ARRAY_DIMENSIONS_ if no HDF5 dimension scales.

Reply:
I have no idea - but it seems like something we ought to eventually cover in a test.

My question was exactly: do we need an else block here? It would, I suppose, be for a scalar value.

Reply:
I reverted back to always having this Zarr attribute. The description in the [xarray documentation](http://xarray.pydata.org/en/latest/internals.html#zarr-encoding-specification) seems to imply it is required.